packages/zowe-explorer/README.md

```diff
@@ -156,6 +154,8 @@ Your token has been successfully invalidated.
 
 - **Open recent members**: Zowe Explorer lets you open a list of members you worked on earlier. You can access the list by pressing `Ctrl`+`Alt`+`R` (Windows) or `Command`+`Option`+`R` (Mac).
 
+- **Insert/overtype mode**: Visual Studio Code v1.96 introduced the ability to toggle between insert and overtype mode by pressing the `Insert` key. Overtype mode can be helpful when working with data sets and members with fixed record lengths.
```

Comment:
I think you need some plurals here: "**Insert/overtype modes**" and "... between insert and overtype modes ... "